### PR TITLE
Print no internet connection as text instead of a dialog box

### PR DESCRIPTION
**Pull request by TheLastProject**
Rambox always starts before my system connects to the Internet. This means that every time I start Rambox I get a dialog box and have to manually reload.

This replaces the dialog box by some simple text in the HTML and auto-reloads every 5 seconds so that Rambox starting early isn't such an issue.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,6 +56,7 @@
 </head>
 
 <body>
+	<div id="noconnection" style="display: block"><h1>No Internet Connection</h1><p>Please, check your internet connection and restart Rambox. If you use a Proxy, please go to Preferences to configure it.</p></div>
```
Comment (StormPooper) [inline index.html]: Shouldn't this be 'display: none' so that it doesn't show until the `isOnline` check returns false?

Comment (TheLastProject) [reply index.html]: You're right, I accidentally committed this to test locally. Fixed now.

File: index.html [hunk 2]
```diff
@@ -8,6 +8,22 @@
 
 	<title>Rambox</title>
 
+	<script type='text/javascript'>
```
Comment (StormPooper) [inline index.html]: This logic appears to have moved here: https://github.com/ramboxapp/community-edition/blob/18e1ed0809bfaea0a750219e814c1d0c3b1ce0ee/app/Application.js#L33

**Comment by TheLastProject** [comment]
Seems the new logic already auto-reloads, so that also solves my main issue. This patch therefore doesn't seem worth rewriting.

**Pull request closed without merging**